lua/mapvote/server/modules/db.lua

```diff
@@ -0,0 +1,35 @@
+local DB = {}
+MapVote.DB = DB
+function DB.CreateTable()
+	sql.Query( "CREATE TABLE IF NOT EXISTS mapvote_played_maps ( map TEXT UNIQUE, play_count INTEGER NOT NULL DEFAULT 0, last_played INTEGER )" )
+end
+
+function DB.MapPlayed(map)
+    if sql.Query( string.format("INSERT OR IGNORE INTO mapvote_played_maps (map) VALUES(%s)", sql.SQLStr(map)) ) == false then
+    	print("MapVote SQLError: ", sql.LastError())
```

Comment:
We should track these
```suggestion
    	error( "MapVote SQLError: " .. sql.LastError() )
```

```suggestion
    	ErrorNoHaltWithStack( "MapVote SQLError: " .. sql.LastError() )
```